Bottom: 251ce87c0cff93e0365c1bd08504d77240b0f0c8
Top:    b6004c656a03f1cecd7bb483178086496ae46a69
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-01-28 14:56:25 -0800

librdmacm: Define ucma_ib_init when IB_ACM is disabled

ucma_ib_init is only defined if IB_ACM is enabled, which is
determined by looking for the infiniband/acm.h header file.
Define ucma_ib_init when IB_ACM is disabled.

Problem reportedy by Suresh Shelvapille <suri@baymicrosystems.com>

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/src/cma.h b/src/cma.h
index bdd6a85..b575b9b 100644
--- a/src/cma.h
+++ b/src/cma.h
@@ -158,15 +158,16 @@ static inline int ERR(int err)
 }
 
 int ucma_init();
-void ucma_ib_init();
 extern int af_ib_support;
 
 #define RAI_ROUTEONLY		0x01000000
 
 #ifdef USE_IB_ACM
+void ucma_ib_init();
 void ucma_ib_cleanup();
 void ucma_ib_resolve(struct rdma_addrinfo **rai, struct rdma_addrinfo *hints);
 #else
+#define ucma_ib_init()
 #define ucma_ib_cleanup()
 #define ucma_ib_resolve(x, y)
 #endif
